source/_patterns/03-templates/page-grid/page-grid.twig

```diff
@@ -20,7 +20,7 @@
               {%- set attributes = attributes|merge({class: attributes.class|merge(['page-grid-item--has-keyline-end']) }) -%}
             {%- endif -%}
 
-            {%- include template|default('templates-content-grid') -%}
+            {%- include template|default('templates-single-column-main-width') -%}
```

Comment:
@thewilkybarkid There are 2 templates `single-column-main-width` and `single-column-full-width`. I'd rather have one template and pass the appropriate CSS class in `attributes` but I couldn't get it to work. YMMV.